zebrad/src/commands/start/sync.rs

```diff
@@ -391,24 +439,35 @@ where
 
             let span = tracing::info_span!("block_fetch_verify", ?hash);
```

Comment:
@yaahc I'm going to try using a warn-level span here, hopefully it's not too verbose.

```suggestion
            // This span is used to help diagnose sync warnings
            let span = tracing::warn_span!("block_fetch_verify", ?hash);
```